watch/app/src/main/java/com/google/samples/quickstart/engagesdksamples/watch/data/converters/ItemToEntityConverter.kt

```diff
@@ -34,22 +38,37 @@ object ItemToEntityConverter {
     val movieBuilder: MovieEntity.Builder =
       MovieEntity.Builder()
         .setName(movie.movieName)
+        .setEntityId(movie.id)
         .addPosterImage(
           Image.Builder()
             .setImageUri(
               Uri.parse("android.resource://" + PACKAGE_NAME + "/" + movie.landscapePoster)
             )
             .setImageWidthInPixel(408)
             .setImageHeightInPixel(960)
+            .setImageTheme(ImageTheme.IMAGE_THEME_UNSPECIFIED)
             .build()
         )
         .setPlayBackUri(Uri.parse(movie.playbackUri))
+        .addPlatformSpecificPlaybackUri(
+          PlatformSpecificUri(Uri.parse(movie.platformSpecificPlaybackUri), movie.platformType)
```

Comment:
Let's use `PlatformSpecificUri.Builder` instead.

Reply:
Changed to use Builder